plugins/plugin.go

```diff
@@ -45,6 +46,7 @@ type Plugin struct {
 	MetricsHost       string
 	Log               *logrus.Entry
 	ConnectorFunc     func([]producers.MetricsMessage, *cli.Context) error
+	Client            *http.Client
```

Comment:
I don't want to export it, but I wanted not to initialize the Client repeatedly and the only way I could figure out to share context was to hold a reference to the client on the Plugin struct.